open_spiel/python/examples/benchmark_games.py

```diff
@@ -57,8 +59,15 @@ def _rollout_until_timeout(game_name, time_limit, give_up_after):
       if len(state.history()) > give_up_after:
         num_giveups += 1
         break
-      action = random.choice(state.legal_actions(state.current_player()))
-      state.apply_action(action)
+      if not state.is_simultaneous_node():
```

Comment:
Nit: It's slightly easier to read when conditions are in positive form.
I would do:
if state.is_simulateneuous_node():
  ...
else:
 ...

Otherwise, the else branch is a double negation, which is harder to read than what I propose.

Reply:
Done.